frontend/public/components/deployment-config.tsx

```diff
@@ -99,6 +105,52 @@ const PauseAction: KebabAction = (kind: K8sKind, obj: K8sResourceKind) => ({
   },
 });
 
+const replicationResource = (kind: string, namespace: string, name: string) => {
+  const resource: WatchK8sResource = {
+    kind,
+    namespace,
+    namespaced: true,
+    selector: {
+      matchLabels: {
+        'openshift.io/deployment-config.name': name,
+      },
+    },
+  };
+  return resource;
+};
+
+const retryRolloutAction = (
+  label: string,
+  kindObj: K8sKind,
+  obj: K8sResourceKind,
+  replicationKind: K8sKind,
+  replicationController: K8sResourceCommon,
+) => {
+  const action: Action = {
+    id: 'retry-rollout',
+    label,
+    cta: () =>
+      retryRollout(replicationKind, replicationController).catch((err) =>
+        errorModal({ error: err.message }),
+      ),
+    insertAfter: 'start-rollout',
+    disabled:
```

Comment:
Also need to disable when the DC is first created and the pods are still coming up
https://user-images.githubusercontent.com/895728/175338809-dc129ffa-2da2-4ade-a25c-d9dcfc44ca66.mov